src/views/SignTransactionLedger.vue

```diff
@@ -175,17 +191,34 @@ export default class SignTransactionLedger extends Vue {
     private async mounted() {
         const network = this.$refs.network as Network;
 
-        let validityStartHeight;
+        let recipient: Nimiq.Address;
+        let value: number;
+        let fee: number;
+        let flags: number;
+        let validityStartHeight: number;
+
         if (this.request.kind === RequestType.SIGN_TRANSACTION) {
             const signTransactionRequest = this.request as ParsedSignTransactionRequest;
+            recipient = signTransactionRequest.recipient;
+            value = signTransactionRequest.value;
+            fee = signTransactionRequest.fee;
+            flags = signTransactionRequest.flags;
             validityStartHeight = signTransactionRequest.validityStartHeight;
```

Comment:
Can be shorter:
```ts
{ recipient, value, fee, flags, validityStartHeight } = signTransactionRequest;
```

Reply:
I think this is already done in a different PR. If not I'll take a look in the final feature branch.